migrations/2024-07-27-212955_add_token_locker_id_in_payment_method/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS network_network_token_locker_id VARCHAR(64) DEFAULT NULL;
```

Comment:
Was this supposed to be `network_token_locker_id`?

```suggestion
ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS network_token_locker_id VARCHAR(64) DEFAULT NULL;
```